report.csv

```diff
@@ -0,0 +1,3 @@
+fruit,quantity
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder
**Place the input and output files into the src/main/resources folder.**